ticdc/troubleshoot-ticdc.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Common Problems and Troubleshooting
+summary: Fix common problems while using TiCDC.
+category: reference
+---
+
+# Common Problems and Troubleshooting
+
+This document includes some common problems that you might have while using TiCDC and provides appropriate solutions. It also summarizes some common operation failures and ways to fix them.
+
+## How to choose start-ts while starting a task
+
+First, you need to know that the `start-ts` of a replication task corresponds to a TSO of the upstream TiDB cluster. The replication task will start its data request from this TSO. Therefore, the `start-ts` should meet two conditions below:
+
+- The value of `start-ts` must be larger than current `tikv_gc_safe_point`, otherwise, an error would occur when creating the task.
```

Comment:
```suggestion
- The value of `start-ts` must be larger than current `tikv_gc_safe_point` value. Otherwise, an error will occurs when you create a task.
```